Review.java

```diff
@@ -165,4 +165,29 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  
+  public static double totalSentiment(String fileName)
+  {
+    String word = "";
+    double senValue;
+    senValue = 0.0;
+    String reviewText = textToString(fileName);
+    for (int i = 0; i < reviewText.length(); i++)
+    {
+     String letter = reviewText.substring(i, i+1);
+     
+     if (!(letter.equals(" ")))
+     {
+      word += letter;
+     }
+     else
+     {
+      removePunctuation(word);
+      senValue += sentimentVal(word);
+      word = "";
+     }
+    }
+    return senValue;
+   }
+
 }

```

Comment:
Everything looks great! Only recommendation is pseudo-code to briefly describe what is going on at each step, that's about it.

Reply:
Thanks!